lib/libutils/ext/include/trace_levels.h

```diff
@@ -23,8 +23,8 @@
  *
  */
 
-#define TRACE_MIN       1
-#define TRACE_ERROR     TRACE_MIN
```

Comment:
maybe update also **trace.c** which start with
```c
#if (TRACE_LEVEL > 0)

#if (TRACE_LEVEL < TRACE_MIN) || (TRACE_LEVEL > TRACE_MAX)
#error "Invalid value of TRACE_LEVEL"
#endif
(...) 
```

Reply:
The new version of the patch include the fix too.